assessment-api/assessment-service/conf/application.conf

```diff
@@ -390,16 +390,16 @@ kafka {
   topic.send.enable : true
   topics.instruction : "sunbirddev.assessment.publish.request"
 }
-objectcategorydefinition.keyspace="category_store"
+objectcategorydefinition.keyspace="dev_category_store"
 question {
-  keyspace = "question_store"
+  keyspace = "dev_question_store"
```

Comment:
remove this change. This is a local change and it will not go as part of the commit